go.mod

```
@@ -0,0 +1,17 @@
+module github.com/okteto/test
+
+go 1.22.2
```

```suggestion
go 1.22.7
```

To use latest update and maybe fix CVEs